task_list/app/models/person.rb

```diff
@@ -0,0 +1,8 @@
+class Person < ActiveRecord::Base
+  has_many :tasks
+
+  def not_complete
```

Comment:
Might recommend renaming this method to be `num_not_complete` or something that more clearly indicates that this method returns the number of incomplete tasks as opposed to the incomplete tasks themselves.